docs/pages/machine-id/guides/github-actions-kubernetes.mdx

```diff
@@ -27,12 +27,13 @@ Actions runners as well as GitHub Enterprise Server.
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
+(!docs/pages/includes/tctl.mdx!)
+
 - A Kubernetes cluster connected to your Teleport cluster. If you do not already
 have one configured, try our
 [Kubernetes Access getting started guide.](../../kubernetes-access/getting-started.mdx)
 In our examples, this Kubernetes cluster will be named `my-kubernetes-cluster`.
-- A local workstation with `tsh` and `tctl` authorized against your Teleport
-cluster as a user with the privileges to create configuration resources.
+- Your user should have the privileges to create token resources.
```

Comment:
Should we clarify here if we're talking about Kubernetes or Teleport, or would it be obvious to the reader?